Source/control.cpp

```diff
@@ -213,33 +213,33 @@ char SpellITbl[] = {
 };
 
 enum panel_button_id {
-	PANBTN_CHARINFO,
-	PANBTN_QLOG,
-	PANBTN_AUTOMAP,
-	PANBTN_MAINMENU,
-	PANBTN_INVENTORY,
-	PANBTN_SPELLBOOK,
-	PANBTN_SENDMSG,
-	PANBTN_FRIENDLY,
+	kPanbtnCharinfo,
+	kPanbtnQlog,
+	kPanbtnAutomap,
+	kPanbtnMainmenu,
+	kPanbtnInventory,
+	kPanbtnSpellbook,
+	kPanbtnSendmsg,
+	kPanbtnFriendly,

```

Comment:
This convention makes zero sense to me for enums (the `k` prefix).

Reply:
See if you like this better: https://github.com/diasurgical/devilutionX/compare/6ebb279ca993d2e75805826f24a6f4ad87437d8d..3eb7f46feab5dea4e9bbcfc2d7ade243f6536fbd

Reply:
Yes please.

Reply:
Should be replaced by enum class anyway.